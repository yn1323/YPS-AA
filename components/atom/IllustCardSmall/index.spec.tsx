import TestWrapper from '@test/TestWrapper'
import { render } from 'enzyme'
import React from 'react'
import { act } from 'react-test-renderer'
import Component from '.'

test(Component.name, async () => {
  let component
  await act(async () => {
    component = await render(
      <TestWrapper>
        <Component
          isGray
          imageUri="/images/functionalIcon1.png"
          text="sampletext"
        />
      </TestWrapper>
    )
  })
  expect(component).toMatchSnapshot()
  await act(async () => {
    component = await render(
      <TestWrapper>
        <Component imageUri="/images/functionalIcon1.png" text="sampletext" />
      </TestWrapper>
    )
  })
  expect(component).toMatchSnapshot()
})
