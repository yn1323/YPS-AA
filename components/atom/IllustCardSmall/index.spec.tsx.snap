// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IllustCardSmall-snapshot 1`] = `
<IllustCardSmall
  imageUri="/images/functionalIcon1.png"
  text="sampletext"
>
  <IllustCardSmall__Container
    isGrayBg={false}
  >
    <div
      className="IllustCardSmall__Container-sc-1kir3jt-0 ggRchh"
    >
      <IllustCardSmall__ImageWrapper>
        <div
          className="IllustCardSmall__ImageWrapper-sc-1kir3jt-1 kEjaWo"
        >
          <Image1
            height="80%"
            objectFit="contain"
            src="/images/functionalIcon1.png"
            width="80%"
          >
            <div
              style={
                Object {
                  "boxSizing": "border-box",
                  "display": "inline-block",
                  "margin": 0,
                  "maxWidth": "100%",
                  "overflow": "hidden",
                  "position": "relative",
                }
              }
            >
              <div
                style={
                  Object {
                    "boxSizing": "border-box",
                    "display": "block",
                    "maxWidth": "100%",
                  }
                }
              >
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iODAiIGhlaWdodD0iODAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                  style={
                    Object {
                      "border": "none",
                      "display": "block",
                      "margin": 0,
                      "maxWidth": "100%",
                      "padding": 0,
                    }
                  }
                />
              </div>
              <noscript>
                <img
                  data-nimg={true}
                  decoding="async"
                  src="/_next/image?url=%2Fimages%2FfunctionalIcon1.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2FfunctionalIcon1.png&w=96&q=75 1x, /_next/image?url=%2Fimages%2FfunctionalIcon1.png&w=256&q=75 2x"
                  style={
                    Object {
                      "border": "none",
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": 0,
                      "left": 0,
                      "margin": "auto",
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                      "minHeight": "100%",
                      "minWidth": "100%",
                      "objectFit": "contain",
                      "objectPosition": undefined,
                      "padding": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                      "width": 0,
                    }
                  }
                />
              </noscript>
              <img
                data-nimg={true}
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": "contain",
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </div>
          </Image1>
        </div>
      </IllustCardSmall__ImageWrapper>
      <IllustCardSmall__TextWrapper>
        <div
          className="IllustCardSmall__TextWrapper-sc-1kir3jt-2 kuiZbm"
        >
          sampletext
        </div>
      </IllustCardSmall__TextWrapper>
    </div>
  </IllustCardSmall__Container>
</IllustCardSmall>
`;

exports[`IllustCardSmall-snapshot 2`] = `
<IllustCardSmall
  imageUri="/images/functionalIcon1.png"
  isGray={true}
  text="sampletext"
>
  <IllustCardSmall__Container
    isGrayBg={true}
  >
    <div
      className="IllustCardSmall__Container-sc-1kir3jt-0 gORLmW"
    >
      <IllustCardSmall__ImageWrapper>
        <div
          className="IllustCardSmall__ImageWrapper-sc-1kir3jt-1 kEjaWo"
        >
          <Image1
            height="80%"
            objectFit="contain"
            src="/images/functionalIcon1.png"
            width="80%"
          >
            <div
              style={
                Object {
                  "boxSizing": "border-box",
                  "display": "inline-block",
                  "margin": 0,
                  "maxWidth": "100%",
                  "overflow": "hidden",
                  "position": "relative",
                }
              }
            >
              <div
                style={
                  Object {
                    "boxSizing": "border-box",
                    "display": "block",
                    "maxWidth": "100%",
                  }
                }
              >
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iODAiIGhlaWdodD0iODAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                  style={
                    Object {
                      "border": "none",
                      "display": "block",
                      "margin": 0,
                      "maxWidth": "100%",
                      "padding": 0,
                    }
                  }
                />
              </div>
              <noscript>
                <img
                  data-nimg={true}
                  decoding="async"
                  src="/_next/image?url=%2Fimages%2FfunctionalIcon1.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2FfunctionalIcon1.png&w=96&q=75 1x, /_next/image?url=%2Fimages%2FfunctionalIcon1.png&w=256&q=75 2x"
                  style={
                    Object {
                      "border": "none",
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": 0,
                      "left": 0,
                      "margin": "auto",
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                      "minHeight": "100%",
                      "minWidth": "100%",
                      "objectFit": "contain",
                      "objectPosition": undefined,
                      "padding": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                      "width": 0,
                    }
                  }
                />
              </noscript>
              <img
                data-nimg={true}
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": "contain",
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </div>
          </Image1>
        </div>
      </IllustCardSmall__ImageWrapper>
      <IllustCardSmall__TextWrapper>
        <div
          className="IllustCardSmall__TextWrapper-sc-1kir3jt-2 kuiZbm"
        >
          sampletext
        </div>
      </IllustCardSmall__TextWrapper>
    </div>
  </IllustCardSmall__Container>
</IllustCardSmall>
`;
